src/pages/resultsView/survival/styles.module.scss

```diff
@@ -0,0 +1,42 @@
+.SurvivalTable {
+  margin: 30px auto;
+  width: 65%;
+}
+
+table {
```

Comment:
i don't think this is nested so this will apply to all <table>?

Reply:
Yes, will make it more specific later, we can merge for now.